e2e-test/trace/src/integrationTest/java/com/amazon/dataprepper/integration/trace/EndToEndRawSpanTest.java

```diff
@@ -1,15 +1,20 @@
 /*
- * Copyright OpenSearch Contributors
- * SPDX-License-Identifier: Apache-2.0
+ *  SPDX-License-Identifier: Apache-2.0
```

Comment:
Please use the shorter license file.

Unless you copied this code from another project, the shorter license file is the correct one.